modules/bishop.py

```diff
@@ -0,0 +1,55 @@
+import copy
+
+from modules.chess_piece import ChessPiece
+
+class Bishop(ChessPiece):
+    def __init__(self, position, color, id):
+        super().__init__(position, "bishop", color, id)
+        self.state["has_moved"] = False
```

Comment:
```suggestion
```
This was necessary for the king to implement calling and the pawn to implement the double-move, but it shouldn't matter for the bishop.